lib/config/go_routes/go_router.dart

```diff
@@ -76,6 +91,13 @@ abstract class AppRoutes {
         const Splash(),
       ),
     ),
+    GoRoute(
+      path: '/userInfoLoading',
+      pageBuilder: (context, state) => defaultPageBuilder(
+        context,
+        UserInfoLoadingPage(userId: Matrix.of(context).client.userID),
```

Comment:
Should get the current persisted active account and get user info of this inside the page